lab/app/config/router-config.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = ['$stateProvider', '$urlRouterProvider', routerConfig];
+
+function routerConfig($stateProvider, $urlRouterProvider) {
+  $urlRouterProvider.when('/', '/home');
```

Comment:
This should be `$urlRouterProvider.when('', '/home');`, for some reason `/` does not work.